tensorflow/buildspec.yml

```diff
@@ -44,6 +62,28 @@
       docker_file: !join [ docker/, *VERSION, /, *DOCKER_PYTHON_VERSION, /, *CUDA_VERSION, /Dockerfile., *DEVICE_TYPE ]
       context:
         <<: *TRAINING_CONTEXT
+    BuildTensorflowCPUInferencePy3DockerImage:
+      <<: *INFERENCE_REPOSITORY
+      build: &TENSORFLOW_CPU_INFERENCE_PY3 false
+      image_size_baseline: 4899
+      device_type: &DEVICE_TYPE cpu
+      python_version: &DOCKER_PYTHON_VERSION py3
+      tag_python_version: &TAG_PYTHON_VERSION py37
+      tag: !join [ "2.2.0", "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION ]
```

Comment:
Nit: define the custom version as a variable, so that it need not be repeated again.